src/index.ts

```diff
@@ -9,13 +9,17 @@ import {
 import { MapSeriesIterator, PluginOptions } from './interfaces';
 
 import ReadCompat from './read_compat';
+import buidlOutputWrapper from 'broccoli-output-wrapper';
```

Comment:
typo in this variable name here

```suggestion
import buildOutputWrapper from 'broccoli-output-wrapper';
```